src/rpcblockchain.cpp

```diff
@@ -706,9 +709,10 @@ UniValue getblockchaininfo(const UniValue& params, bool fHelp)
     bip9_softforks.push_back(BIP9SoftForkDesc("segwit", consensusParams, Consensus::DEPLOYMENT_SEGWIT));  // MVF-BU added
     obj.push_back(Pair("softforks",             softforks));
     obj.push_back(Pair("bip9_softforks", bip9_softforks));
+    obj.push_back(Pair("difficultyadjinterval", consensusParams.DifficultyAdjustmentInterval(tip->nHeight))); // MVF-BU
 
     // MVF-BU begin output hardfork description (MVHF-BU-DES-TRIG-9)
-    if (!isMVFHardForkActive)
+    if (!isMVFHardForkActive && consensusParams.MVFisWithinRetargetPeriod(tip->nHeight))
```

Comment:
I find it difficult to see how these two can be true at the same time.
Shouldn't it be an OR condition?